.NET/Microsoft.Recognizers.Text.DateTime/Utilities/MergedParserUtil.cs

```diff
@@ -520,6 +520,39 @@ public static List<DateTimeParseResult> DateTimeResolutionForSplit(DateTimeParse
             return results;
         }
 
+        public static DateTimeParseResult TasksModeModification(DateTimeParseResult slot, DateObject referenceTime)
```

Comment:
Code comment here with a general sense of the types of modifications?

Reply:
added code comment.